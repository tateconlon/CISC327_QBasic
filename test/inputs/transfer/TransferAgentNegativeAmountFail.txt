login
agent
transfer
1234567
2234567
-5
logout