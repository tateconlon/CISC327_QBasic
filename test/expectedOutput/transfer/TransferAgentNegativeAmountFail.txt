login as agent successful
Cannot transfer less than 1 cent
Transaction summary written and system logged out successfully